src/main/java/com/olgaboiar/mint/Response.java

```diff
@@ -0,0 +1,30 @@
+package com.olgaboiar.mint;
+
+public class Response {
+    private Header header;
+    private Body body;
+    String responseToSend;
```

Comment:
Just a small cleanup suggestion, but the `responseToSend` is only ever used locally in the `prepareResponse()` method. It appears that it would be safe to remove from the scope of the instance.